pkg/signalmeow/upgrade.go

```diff
@@ -191,3 +191,23 @@ func upgradeV3(tx *sql.Tx, _ *StoreContainer) error {
 	}
 	return nil
 }
+
+func upgradeV4(tx *sql.Tx, _ *StoreContainer) error {
+	_, err := tx.Exec(`
```

Comment:
This needs to check the dialect in StoreContainer and not do anything on sqlite

Reply:
Done by 6f378d1